chainer/functions/array/rollaxis.py

```diff
@@ -0,0 +1,60 @@
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Rollaxis(function.Function):
+
+    """Roll axis of an array."""
+
+    def __init__(self, axis, start):
+        self.axis = int(axis)
```

Comment:
Is there any reason why you apply `int` function to `axis`and `start`? Document imposes `axis` and `start` be integers. So we need not to convert them.